### PR TITLE
Adding fromJson to OutcomeCohortExpression; alignment to specifications

### PR DESCRIPTION
**Pull request by anthonysena**
Updates for serializing negative control outcome expressions from JSON and aligning the property names to the specification.

### CODE REVIEW

File: src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java [hunk 1]
```diff
@@ -16,17 +17,16 @@
...
-		try {
-			CohortExpression expression = JSON_MAPPER.readValue(json, CohortExpression.class);
-			return expression;
-		} catch (Exception e) {
-			throw new RuntimeException("Error parsing cohort expression", e);
-		}
-	}
- */
+ public static OutcomeCohortExpression fromJson(String json) {
+   try {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java]: https://github.com/OHDSI/StandardizedAnalysisAPI/blob/pathways-analysis/src/main/java/org/ohdsi/analysis/Utils.java#L67

Comment (anthonysena) [reply src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java]: That makes sense to me @pavgra; however, I'm not sure if we're leveraging SAA in circe just yet? I'll leave it to @chrisknoll's discretion on how to proceed.

Comment (chrisknoll) [reply src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java]: Post Symposium.

Comment (anthonysena) [reply src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java]: To make sure I'm clear: we can refactor to use the SAA Utils for formatting after symposium. Can we merge this change in with that understanding? I see @pavgra linked it to the WebAPI issue. I need this functionality for Symposium so if we can merge it in, that would be greatly appreciated. @schuemie needs this for testing on Hydra.

Comment (chrisknoll) [reply src/main/java/org/ohdsi/circe/cohortdefinition/negativecontrols/OutcomeCohortExpression.java]: Ok, it will be done.

**Pull request merged**